src/rating/Rating.js

```diff
@@ -245,6 +293,23 @@ const styles = StyleSheet.create({
   },
 });
 
+const fractionsType = (props, propName, componentName) => {
+  if (props[propName]) {
+    const value = props[propName];
+    if (typeof value === 'number') {
+      return value >= 0 && value <= 20
+        ? null
+        : new Error(
+            `\`${propName}\` in \`${componentName}\` must be between 0 and 20`
```

Comment:
Yeah, I'm probably overthinking it. This is for (accidental) invalid `fractions` values, like `-1` or `true`. The red screen of death only says you're using `toFixed` wrong, which I imagine will leave some users scratching their heads. This way you also get a message in the terminal:
![screen shot 2017-06-05 at 15 09 06](https://cloud.githubusercontent.com/assets/241553/26781698/075ac9c8-4a01-11e7-9190-39f898862d97.png)

Not passing `fractions` works as expected :)